Alignment/MillePedeAlignmentAlgorithm/scripts/mps_runPede_rfcp_template.sh

```diff
@@ -182,10 +182,12 @@ gzip -f millepede.eve
 gzip -f millepede.mon
 
 #list IOVs
-for tag in $(sqlite3 alignments_MP.db  "SELECT NAME FROM TAG;")
-do
-    conddb --db alignments_MP.db list ${tag}
-done
+conddb --db alignments_MP.db list Alignments
+conddb --db alignments_MP.db list Deformations
+conddb --db alignments_MP.db list SiStripLorentzAngle_deco
+conddb --db alignments_MP.db list SiStripLorentzAngle_peak
+conddb --db alignments_MP.db list SiPixelLorentzAngle
+conddb --db alignments_MP.db list SiStripBackPlaneCorrection
```

Comment:
@prigoll you are reintroducing old code here; the for loop is the more general and cleaner version of the code you are reintroducing here